poolrpc/trader.proto

```diff
@@ -78,6 +78,13 @@ service Trader {
     rpc RecoverAccounts (RecoverAccountsRequest)
         returns (RecoverAccountsResponse);
 
+    /* pool: `accounts accountmodificationfees`
```

Comment:
The command in the back ticks needs to match an actual CLI sub command. It is parsed by the API docs generator and will fail since the command is called `listfees` instead of `accountmodificationfees`.